app/models/like.rb

```diff
@@ -1,4 +1,11 @@
 class Like < ApplicationRecord
-  belongs_to :users
-  belongs_to :posts
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+  after_save :update_likes_counter
+
+  # Method to update the likes counter for a post.
+  def update_likes_counter
+    post = Post.find_by(id: post_id)
+    post.increment!(:likes_counter)
+  end
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on making this method because it is working and using the increment method as suggested by the previous reviewer 👍 but it would be nice if you make this method more concise if you don't write the line to find the Post id because this method will automatically find which post has a like 😊 So it would look like below and you can also make the other increment methods like the below one ⬇️

```ruby
  def update_likes_counter
    post.increment!(:likes_counter)
  end
```